pages/login/index.js

```diff
@@ -67,6 +65,14 @@ export default function Profile() {
     <>
       {!formSelect && (
         <StyDiv>
+          <StyImage
+            priority={true}
+            src={logo}
+            width={300}
+            height={200}
+            style={{ objectFit: "contain" }}
+            alt="Beanerd Logo"
+          ></StyImage>
```

Comment:
A closing tag should not be needed. As you already use a styled component I  would prefer to not use any additional inline stylings.

Reply:
The style isnt even needed! I did not know, why I put it there.